match/lib/match/storage/s3_storage.rb

```diff
@@ -36,12 +40,18 @@ def self.configure(params)
           UI.message("The above is just a warning, fastlane will continue as usual now...")
         end
 
+        if (s3_endpoint.to_s.length > 0) && !s3_force_path_style
+          UI.message("When you use a custom S3 endpoint, in most cases you also need to set `force_path_style` to `true`")
+        end
```

Comment:
```suggestion
        if !s3_endpoint&.empty? && !s3_force_path_style
          UI.message("When you use a custom S3 endpoint, in most cases you also need to set `force_path_style` to `true`")
        end
```

This will be `true` for non-empty strings and `false` for either empty strings or `nil` 😃 

Perhaps also explain why in the message? (Following along the better description for this option as I mentioned above)

Reply:
From my own experience using S3-compatible storages with Ruby S3 SDK, these two params should be both set, at least for Minio and CloudFlare R2.

I removed this message first. Maybe it is not needed. Users should be able to know the details while using S3